cmd/awat/translation.go

```diff
@@ -113,12 +115,19 @@ var startTranslationCmd = &cobra.Command{
 		server, _ := cmd.Flags().GetString(serverFlag)
 		awat := model.NewClient(server)
 
+		translationType, _ := cmd.Flags().GetString(translationTypeFlag)
+		if translationType != model.MattermostWorkspaceBackupType &&
+			translationType != model.SlackWorkspaceBackupType {
+			return fmt.Errorf("unknown Translation type \"%s\" provided", translationType)
```

Comment:
```suggestion
			return errors.Errorf("unknown Translation type %q provided", translationType)
```

Reply:
:facepalm:

Reply:
oh I see what happened, I just had `errors` imported instead of `github.com/pkg/errors` so `errors.Errorf` didn't exist and I got confused :stuck_out_tongue_closed_eyes: